src/renderer/tvgFrameModule.h

```diff
@@ -38,7 +38,9 @@ class FrameModule: public ImageLoader
     virtual float totalFrame() = 0;         //return the total frame count
     virtual float curFrame() = 0;           //return the current frame number
     virtual float duration() = 0;           //return the animation duration in seconds
-
+    virtual float startFrame() { return 0.0f; }; //return start frame
+    virtual bool getSegment(float& beign, float& end, const char* marker = nullptr) { return false; }
```

Comment:
The marker is a lottie specific concept. It should not be in this FrameModule interface. You might wanna add add those necessary interfaces in the LottieLoader